docs/versioned/spark/spark-writes.md

```diff
@@ -98,6 +98,11 @@ WHEN NOT MATCHED AND s.event_time > still_valid_threshold THEN INSERT (id, count
 
 Only one record in the source data can update any given row of the target table, or else an error will be thrown.
 
+!!! Note
+    By default Spark uses copy-on-write merge mode.
+    With spark-3.2 and onwards, iceberg supports merge-on-read mode.    
```

Comment:
Not sure what everyone thinks, but seems there's not much context.  Should we explain what it means (that it writes v2 delete files)?